Library/Homebrew/github_packages.rb

```diff
@@ -301,12 +301,21 @@ def upload_bottle(user, token, skopeo, formula_full_name, bottle_hash, keep_old:
       formula_annotations_hash = image_index["annotations"]
       manifests = image_index["manifests"]
     else
+      # Use official license annotation if within GitHub's 256 character limit
+      # TODO: Consider generating an abbreviated license when over the limit
+      license = bottle_hash["formula"]["license"]
+      license_annotation_key = if license.length <= 256
+        "org.opencontainers.image.licenses"
+      else
+        "org.opencontainers.image.license"
+      end
+
       formula_annotations_hash = {
         "com.github.package.type"                => GITHUB_PACKAGE_TYPE,
         "org.opencontainers.image.created"       => created_date,
         "org.opencontainers.image.description"   => bottle_hash["formula"]["desc"],
         "org.opencontainers.image.documentation" => documentation,
-        "org.opencontainers.image.license"       => bottle_hash["formula"]["license"],
```

Comment:
I kinda debate if we should always set this field but also set `licenses` only if we can do it without going over limit. Thoughts?